### PR TITLE
feat(comps): amend default apps

### PR DESCRIPTION
**Pull request by madonuko**

### CODE REVIEW

**Comment by ninelore** [comment]
if not done already: replace gnome-terminal with console since former is gtk3 and latter is the only one mentions at [upstream gnome](https://apps.gnome.org/)

**Comment by korewaChino** [comment]
> replace gnome-terminal with console since former is gtk3 and latter is the only one mentions at [upstream gnome](https://apps.gnome.org/)

Fedora actually uses ptyxis by default now so probably no need?

**Comment by ninelore** [comment]
yea fair enough

**Comment by nothingneko** [comment]
yeah we didn't switch to Console because it cut so many features, now that Ptyxis is being pushed so hard I'm finally comfortable switching

**Review by nothingneko — DISMISSED** [review]

**Comment by madonuko** [comment]
Do we switch to ptyxis?

**Comment by lleyton** [comment]
we should

File: comps.xml [hunk 1]
```diff
@@ -180,8 +181,8 @@
       <packagereq type="default">kolourpaint</packagereq>
       <packagereq type="default">kate</packagereq>
       <packagereq type="default">kmail</packagereq>
-      <packagereq type="default">vlc</packagereq>
```
Comment (nothingneko) [inline comps.xml:183]: i chose these two for a reason, kalk is a lot simpler and nicer looking than kcalc, and vlc has been requested so many times and makes sense in this edition

Comment (madonuko) [reply comps.xml:183]: For some reason my installation doesn't come with `kalk`? and `vlc` doesn't work well on plasma with my experience…

Comment (nothingneko) [reply comps.xml:183]: hmmm let's merge your changes so we can be in line with upstream but play with the other options

**Review by lleyton — APPROVED** [review]

**Pull request merged**